### ISSUE TITLE
Ability to save each run to compare progress

### ISSUE THREAD

**Issue by evitalis** [topeterk/HitCounterManager#14]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Some way to export each run in order to import into another application or show a graph of progress.

**Describe alternatives you've considered**
Screenshotting the individual runs, or manually entering the data into a CSV file externally.

**Additional context**
The application already tracks what number attempt someone may be on. I would simply like to see the progress outside of just the PB. Say run 4 is a PB but 5 and 6 are worse, I find value in still seeing this. It will help see if one split is getting worse or the run overall.

Thanks for the hard work on this, especially the Linux compatibility.

**Comment by topeterk** [comment]
Hi, yes I think this could be a great idea to keep track of the personal progress.
The only difficulty I'm seeing is the split modification: In case one adds/move/remove splits (or even just edit the split name) it is hard to tell which split corresponds to which (previous) split in order to maintain a comparable history. Would be pretty nice to have a solution for this, otherwise the graphs are less accurate.
However one may start by ignoring this, so someone who is not editing the splits will see a comparable history anyway. And when the run gets changed then there is a "break" in the history but that eventually may not matter at the end of the day.

**Comment by evitalis** [comment]
Yeah I did some thinking on this and you are right. Each split might be too much so perhaps just keeping the totals for each run would be fine.

**Comment by UkuleleAversion** [comment]
I and another runner I was speaking to today was wondering if there was a way you could do this. Seeing the split data from many full runs in a spreadsheet would be invaluable. All we'd need is the split names and hit totals and PBs. 

Also, renaming/removing of splits isn't a big deal. Besides, if users of HCM know they want to track their progress in this way, they'd be more conservative about renaming/removing splits.
